### ISSUE TITLE
[BUG]: No Elite Rating changes

### ISSUE THREAD

**Issue by lisbetoo** [OoliteProject/oolite#486]
I haven't gotten a single change to my elite rating, despite killing some pirates.  The count is still zero, even though I've been credited with the kills through OXPs, like the RRS mission I was on or the email system. I have, I hope, attached the log files, a text file list of my current expansion set, and a text file for my saved game.

I am running Oolite 1.90 on Windows 10
Processor:	11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz   2.30 GHz
Installed RAM:	16.0 GB (15.8 GB usable)
System type:	64-bit operating system, x64-based processor
Graphics:          NVIDIA GeForce RTX 3050 Laptop GPU

Can't find the attachments.  Can't put them inline- too long for the post.  If you can't find them either, please let me know how to get you the files you need.

**Comment by AnotherCommander** [comment]
Kill count works fine in my game. If you don't see it increasing then it has to be due to OXP interference. You will have to find out which OXP is causing it and then either notify the OXP author about the problem or remove the offending OXP.

As a first test, you can try removing all OXPs and just playing the core game for a while. You should see your kill count increasing normally and your rating changing to Mostly Harmless once you hit your 8th kill.

**Comment by lisbetoo** [comment]
You're right, I found that strict mode works.  However, I have well over a hundred OXPs installed, none of which I would expect to interfere with the game's count. I would very much like to not remove and replace one by one.  Are there any suggestions as to where to start finding the breakage?  Even a category would help.

**Comment by lisbetoo** [comment]
Found the OXP.  For anyone who also has this issue, GalCop's Most Wanted and/or Bounty System has a set of parameters available through the OxpConfig OXP.  These parameters can _defer bounty payments and elite rating increase_ until you go to the GalCop office though the F4 screen.  And the default settings were to both defer, and require that you pick up your bounties within the same system you earned them in.  I would call the wiki pages unclear on this component of the OXPs; I saw information on picking up a bonus bounty, but nothing on deferring receipt of the basic bounty or the elite rating.  Or maybe I'm not reading carefully enough.  Either way, here's the information for the next person with the issue.

**Comment by phkb** [comment]
The "issue" is with the Bounty System OXP. A recent change to that OXP means that you need to dock and visit the GalCop Security office in order for your bounty claims to be ratified. Not a bug, but a new feature.

**Comment by lisbetoo** [comment]
Many thanks.  Talking here instead of trying to deal with the board.

I don't know how possible, or compatible with your narrative, this idea is but I think a reminder email would be helpful. I can easily see a commander jumping out after a fight and not having the time (or perhaps interest) to immediately backtrack and get their bounties.  An email reminding a commander with X deferred bounties, or one who has a bounty that has been deferred for Y hours to stop by GalCop and pick up their bounties would prevent them from getting lost.  Especially if it can give a list of systems necessary to visit in order to do so, given the current config setting.

Also, I note that the deferral is not comprehensive.  I actually figured out I should be looking at these OXPs because I got two points towards Elite, went through the save file and the last autosave looking for information since I had a number to work with rather than a zero and noticed "deferred" next to "BountySystem."  I can't give you the save file, I overwrote it, and I'm pretty sure the logs, in the process of extracting my elite rating in it's entirety (all 32 points of it), but if it happens again I'm happy to share either or both with you if you have a use for them.
